{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/js/navigation/navigation.js

```diff
@@ -20,34 +20,36 @@ export default class Navigation {
     this.setupListeners()
   }
 
+  /**
+   * Cache any dom elements we will use multiple times.
+   */
   setUpElements () {
     const el = document.querySelector('.nav-Header')
 
     this.els = {
       el,
       document: document.documentElement || document.body,
       header: document.querySelector('.hd-Header'),
-      nav: document.querySelector('.nav-Header_Items')
+      nav: document.querySelector('.nav-Header_Items'),
+      logo: document.querySelector('.hd-Header_Logo')
     }
   }
 
+  /**
+   * Cache any elements for the mobile nav that need to be cached after the mobile nav is created.
+   */
   setUpMobileElements () {
-    this.els.trigger = this.els.el.querySelector('.nav-Header_Trigger')
+    this.els.trigger = document.querySelector('.nav-Header_Trigger')
 
     this.els.backdrop = this.els.document.querySelector('.nav-Header_Backdrop')
-
-    this.els.tops = this.els.document.querySelectorAll(
-      `.nav-Header_Items,
-      .nav-Header_Backdrop,
-      .nav-Header_Dropdown,
-      .nav-Mobile`
-    )
   }
 
+  /**
+   * If we're on mobile we want to track the selected states to control the sub navigation open
+   * / closed. aria-selected won't be used on lg++ so we programmatically add it just for sizes
```

Comment:
We shouldn't be using `aria-` attributes at all; the entire mobile nav is `aria-hidden="true"` so it makes no sense.

Reply:
It get's used for the styling of the current active items, as it's also `aria-selected` for desktop it kind of made sense to me to just leave it done like this for mobile too, I could replace with a modifier class for handling active items for mobile though if you'd prefer?

Reply:
Actually my bad was getting aria-current and aria-selected mixed up ignore the above 😁